### DISCUSSION TITLE
Unlimited Memory?

### DISCUSSION THREAD

**Discussion by Heywaht** [itzg/docker-minecraft-server#2789] [Q&A] [answered]
Hi, is it possible to allocate unlimited ram (memory) with the -e MEMORY tag? 
That you don't have to allocate a GB. The server itself should decide how much ram it should use and if it is no longer sufficient then crash.

**Comment by itzg** [comment, marked as answer]
That's really a general Java question

https://stackoverflow.com/questions/39717077/how-do-i-start-a-jvm-with-unlimited-memory

This section includes a note about using percentage calculated heap init/max sizing with this image 

https://docker-minecraft-server.readthedocs.io/en/latest/configuration/jvm-options/#memory-limit
